src/main/java/com/abc/hublearning/web/dto/StudentDto.java

```diff
@@ -0,0 +1,24 @@
+package com.abc.hublearning.web.dto;
+
+import java.util.List;
+
+public class StudentDto {
+    private List<Long> coursesIds;
```

Comment:
ordinea fieldurilor in clasa, sa lucram pe curat.

Reply:
Done